pontoon/base/checks.py

```diff
@@ -0,0 +1,114 @@
+import os
```

Comment:
Hi @Pike,  This PR still needs some work. I would really appreciate if you could look at things related to compare-locales.